Makefile

```diff
@@ -56,7 +56,7 @@ provider::
 	go install -ldflags "-X github.com/pulumi/pulumi-${PACK}/pkg/version.Version=${VERSION}" ${PROJECT}/cmd/${PROVIDER}
 
 lint::
-	golangci-lint run
+	#golangci-lint run
```

Comment:
Is this on purpose?

Reply:
Yes it’s been taken out if all for now - issue in Pulumi/Pulumi about it